hbp_knowledge/er-stress/guo2019.bel

```diff
@@ -0,0 +1,374 @@
+p(CONSO:"Tau isoform F (441 aa)",frag("187_441")) -> path(MESH:"Motor Disorders")
+p(CONSO:"Tau isoform F (441 aa)",frag("187_441")) neg p(MESH:Proteins, loc(GO:"synaptic membrane"))
+
+SET Evidence = "Lysates of CHO cells stably expressing either
+full-length (2N4R isoform) human tau (CHO-FL) or
+Tau35 (CHO-Tau35) were analyzed on western blots
+using antibodies to phosphorylated tau (AT8, pSer202/
+pThr205; AT180, pThr231; and PHF1, pSer396/pSer404)
+and total tau (Fig. 1a). Antibodies recognizing phosphorylated
+tau exhibited only modest (AT180, PHF1) or negligible
+(AT8) immunoreactivity with FL-tau, whereas all
+tau antibodies recognized multiple Tau35 bands (Fig.
+1a)."
+#FL-tau - 2N4R tau
+p(CONSO:"Tau isoform F (441 aa)", pmod(Ph,Thr,231))
+p(CONSO:"Tau isoform F (441 aa)", pmod(Ph,Ser,369))
+p(CONSO:"Tau isoform F (441 aa)", pmod(Ph,Ser,404))
+p(CONSO:"Tau isoform F (441 aa)",frag("187_441"), pmod(Ph,Ser,202))
+p(CONSO:"Tau isoform F (441 aa)",frag("187_441"), pmod(Ph,Thr,205))
+p(CONSO:"Tau isoform F (441 aa)",frag("187_441"), pmod(Ph,Thr,231))
+p(CONSO:"Tau isoform F (441 aa)",frag("187_441"), pmod(Ph,Ser,396))
+p(CONSO:"Tau isoform F (441 aa)",frag("187_441"), pmod(Ph,Ser,404))
+
+SET Evidence = "Expression of FL-tau induced formation of
+microtubule bundles and robust co-localization of tau
+with microtubules, neither of which were apparent in
+CHO-Tau35 cells (Fig. 2a)."
+p(CONSO:"Tau isoform F (441 aa)") -> bp(GO:"microtubule bundle formation")
+p(CONSO:"Tau isoform F (441 aa)") -> p(CONSO:"Tau isoform F (441 aa)", loc(GO:"microtubule bundle"))
+
+SET Evidence = "Colocalization analyses of the FL-tau/
+α-tubulin and Tau35/α-tubulin immunofluorescence in
+the two cell lines demonstrated a significantly reduced
+correlation of Tau35 with microtubules in CHO cells
+(Fig. 2b, P < 0.01)."
+#C129386 - alpha tubulin
+p(CONSO:"Tau isoform F (441 aa)",frag("187_441")) -| composite(p(CONSO:"Tau isoform F (441 aa)",frag("187_441")), p(NCIT:C129386))
+
+SET Evidence = "Thus, unlike FL-tau, Tau35 localizes
+predominantly in the cytosol, rather than binding to
+and stabilizing microtubules."
+p(CONSO:"Tau isoform F (441 aa)",frag("187_441"), loc(GO:cytosol))
+p(CONSO:"Tau isoform F (441 aa)",frag("187_441")) -| act(p(CONSO:"Tau isoform F (441 aa)",frag("187_441")), ma(GO:"microtubule binding"))
+p(CONSO:"Tau isoform F (441 aa)",frag("187_441")) -| bp(GO:"microtubule cytoskeleton organization")
+
+SET Evidence = "Only a minor
+proportion of the microtubule population was acetylated
+in CHO-Tau35 and CHO cells (Fig. 2e), whereas expression
+of FL-tau resulted in the formation of an ordered
+array of microtubules in CHO cells, and acetylated
+α-tubulin largely overlapped with microtubules detected
+by α-tubulin antibody (Fig. 2e)."
+#Acteylated Lys40 alpha-tubulin
+#C17421 - CHO cells
+p(CONSO:"Tau isoform F (441 aa)",frag("187_441")) -> p(NCIT:C129386, pmod(Ac,Lys,40))
+p(CONSO:"Tau isoform F (441 aa)") => p(NCIT:C129386, pmod(Ac,Lys,40))
```

Comment:
please use NCIT terms only if no other namespace provides a definition;
e.g., for  alpha tubulin IPR002452 works better